hubmap/frontend/src/components/home/HumanSvgComponent.js

```diff
@@ -98,140 +58,23 @@ class HumanAnatomyCard extends React.Component {
-	      this.genetissueArray.forEach((genetissuepair) => {
-	        // index = this.genetissueArray.findIndex((e) => e.tissue === genetissuepair.tissue);
-	        // if(index !== -1)
-	        /* {
-						console.log(datatooltip, index)
-						datatooltip = this.genetissueArray[index];
-					 }
-					 else
-					 {
-						 datatooltip = 'No data available';
-					} */
-	        switch (genetissuepair.tissue.toLowerCase()) {
-	          case 'pancreas':
-	            pancreas.setAttributeNS(null, 'data-tooltip-text', ` ${genetissuepair.tissue}, # genes: ${genetissuepair.genescount}`);
-	            break;
-	          case 'liver':
-	            liver.setAttributeNS(null, 'data-tooltip-text', ` ${genetissuepair.tissue}, # genes: ${genetissuepair.genescount}`);
-	            break;
-	          case 'lung':
-	            lung.setAttributeNS(null, 'data-tooltip-text', ` ${genetissuepair.tissue}, # genes: ${genetissuepair.genescount}`);
-	            break;
-
-	          case 'abdomen':
-	            abdomen.setAttributeNS(null, 'data-tooltip-text', ` ${genetissuepair.tissue}, # genes: ${genetissuepair.genescount}`);
-	            break;
-
-	          case 'kidney':
-	            kidney.setAttributeNS(null, 'data-tooltip-text', ` ${genetissuepair.tissue}, # genes: ${genetissuepair.genescount}`);
-	            break;
-
-	          case 'spleen':
-	            spleen.setAttributeNS(null, 'data-tooltip-text', ` ${genetissuepair.tissue}, # genes: ${genetissuepair.genescount}`);
-	            break;
-
-	          case 'small_intestine':
-	            smallIntestine.setAttributeNS(null, 'data-tooltip-text', ` ${genetissuepair.tissue}, # genes: ${genetissuepair.genescount}`);
-	            break;
-
-	          case 'large_intestine':
-	            largeIntestine.setAttributeNS(null, 'data-tooltip-text', ` ${genetissuepair.tissue}, # genes: ${genetissuepair.genescount}`);
-	            break;
-
-	          case 'bladder':
-	            bladder.setAttributeNS(null, 'data-tooltip-text', ` ${genetissuepair.tissue}, # genes: ${genetissuepair.genescount}`);
-	            break;
-
-	          case 'heart':
-	            heart.setAttributeNS(null, 'data-tooltip-text', ` ${genetissuepair.tissue}, # genes: ${genetissuepair.genescount}`);
-	            break;
-	          default:
-	            break;
-	        }
-	      });
+	      pathTriggers[i].addEventListener('mousemove', utils.showToolTip);
+	      pathTriggers[i].addEventListener('mouseout', utils.hideToolTip);
+		  pathTriggers[i].setAttributeNS(null, 'data-tooltip-text', ` ${pathTriggers[i].id},No gene data available`);
+		  this.experimentState.response.slice(0, this.experimentState.response.length - 1).forEach((experiment) => {
+			  if (pathTriggers[i].id.toLowerCase() === experiment.tissue.name.toLowerCase()) {
+				 pathTriggers[i].setAttributeNS(null, 'data-tooltip-text', ` ${experiment.tissue.name},# genes: ${experiment.genes !== undefined ? experiment.genes.length : 0}`);
```

Comment:
use the attribute setting util you'll create, and space after comma:
```suggestion
				 pathTriggers[i].setAttributeNS(null, 'data-tooltip-text', ` ${experiment.tissue.name}, # genes: ${experiment.genes !== undefined ? experiment.genes.length : 0}`);
```